socketcan_bridge/CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
 cmake_minimum_required(VERSION 2.8.3)
+add_compile_options(-std=c++11)
```

Comment:
C++11 shall not be used!

Reply:
Sigh. It's 2018 i.e. 2 more standards later.

Reply:
c++14 is now the default for melodc :tada: